command/agent/consul/service_client.go

```diff
@@ -2052,18 +2056,6 @@ func isNomadCheck(id string) bool {
 	return strings.HasPrefix(id, nomadCheckPrefix)
 }
 
-// isOldNomadService returns true if the ID matches an old pattern managed by
-// Nomad.
-//
-// Pre-0.7.1 task service IDs are of the form:
```

Comment:
Note to reviewers: I was touching all the callers of this anyway... I think it's probably safe to remove pre-0.7.1 backwards compatibility code for service IDs at this point.